cmd/print.go

```diff
@@ -100,6 +101,31 @@ func printByteInfo(l int) string {
 	return fmt.Sprintf("%v %s", length.value, length.name)
 }
 
+func printSearchResult(i int, result core.Result) {
+	white := color.New(color.FgWhite).SprintFunc()
+	green := color.New(color.FgGreen).SprintFunc()
+	// NOTE: in the future we need to switch based on result.Type
```

Comment:
try to use `TODO` for anything that we need to do moving forward. codeclimate picks up on `TODO` automatically & helps us keep track of tech debt.